CONTRIBUTING.md

```diff
@@ -0,0 +1,137 @@
+# Contributing Guide
+
+Thank you for deciding to contribute and help us improve Percona Distribution for MySQL documentation!
+
+We welcome contributors from all users and community. By contributing, you agree to the [Percona Community code of conduct](https://github.com/percona/community/blob/main/content/contribute/coc.md).
```

Comment:
I suggest to use the rendered online version instead: https://percona.community/contribute/coc/

Reply:
Fixed

Reply:
Somehow that change does not seem to have made it into your last push?

Reply:
Strange. Now it must look OK